containers/Dockerfile.buildenv

```diff
@@ -148,34 +152,16 @@ RUN wget https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.7.tar.gz -O bla
     && cd ../.. \
     && rm -rf yaml-cpp*
 
-# Install include-what-you-use
-# We patch it to allow cyclic includes in boost
-WORKDIR /work
-RUN wget https://github.com/include-what-you-use/include-what-you-use/archive/clang_5.0.tar.gz \
-    && tar -xzf clang_5.0.tar.gz \
-    && rm clang_5.0.tar.gz \
-    && mkdir /work/include-what-you-use-clang_5.0/build \
-    && cd /work/include-what-you-use-clang_5.0/ \
-    && sed -i 's^\\\"third_party/^<boost/^' iwyu_include_picker.cc \
-    && cd /work/include-what-you-use-clang_5.0/build \
-    && cmake -D CMAKE_CXX_COMPILER=clang++-5.0 \
-        -D CMAKE_C_COMPILER=clang-5.0 \
-        -D IWYU_LLVM_ROOT_PATH=/usr/lib/llvm-5.0 .. \
-    && make $PARALLEL_MAKE_ARG \
-    && make install \
-    && cd /work \
-    && rm -rf /work/include-what-you-use-clang_5.0
-
 # Download and build the Charm++ version used by SpECTRE
 # We build both Clang and GCC versions of Charm++ so that all our tests can
 # use the same build environment.
 WORKDIR /work
 ARG CHARM_GIT_TAG=v6.8.0
 # Charm doesn't support compiling with clang-5 without symbolic links
-RUN ln -s $(which clang++-5.0) /usr/local/bin/clang++ \
-    && ln -s $(which clang-5.0) /usr/local/bin/clang \
-    && ln -s $(which clang-format-5.0) /usr/local/bin/clang-format \
-    && ln -s $(which clang-tidy-5.0) /usr/local/bin/clang-tidy
+RUN ln -s $(which clang++-10) /usr/local/bin/clang++ \
+    && ln -s $(which clang-10) /usr/local/bin/clang \
+    && ln -s $(which clang-format-10) /usr/local/bin/clang-format \
+    && ln -s $(which clang-tidy-10) /usr/local/bin/clang-tidy
```

Comment:
the comment above indicates these symlinks are for compiling with `clang-5`. are they still needed for clang-10? if so, please update the comment.